arm-analysisservices/2017-06-09/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
-          "$ref": "#/definitions/BackupConfiguration"
+        "BackupBlobContainerUri": {
+          "type": "string",
+		  "description": "The container URI of backup blob."
```

Comment:
minor: could you please format  json with 2 spaces as the all other things are formatted as 2 spaces in this swagger.